### PR TITLE
Functionality to CreateCluster, ViewCluster, Search

### PR DESCRIPTION
**Pull request by marden-erin**

### CODE REVIEW

File: client/src/components/complex-components/SearchBar/Search.tsx [hunk 1]
```diff
@@ -1,55 +1,36 @@
...
-
   return (
-    //<PageWrapper pageTitle="Search Results" header="Search Results">
     <>
-      <form onSubmit={loadData}>
+      <form>
         <SearchBarWrapper>
           <ThinInput
-            placeholder="Search for authors, books, clusters, or users"
+            placeholder={'Search for books...'}
```
Comment (hieronymus-jodi) [inline client/src/components/complex-components/SearchBar/Search.tsx:23]: Good change 👍

File: client/src/components/simple-components/ButtonsLinks.tsx [hunk 2]
```diff
@@ -59,6 +59,14 @@ const SmallRoundedButton = styled.button<{ color?: ColorType }>`
     ${(props) => ColorCss(props.color)}
 `;
 
+const LargeHalfRoundedButton = styled.button<{ color?: ColorType }>`
+  ${ButtonCss}
+  ${HalfRoundedCss}
+${LargeCss}
+
+${(props) => ColorCss(props.color)}
```
Comment (hieronymus-jodi) [inline client/src/components/simple-components/ButtonsLinks.tsx]: Looks like you need to run prettier

File: client/src/constants/Fonts.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import { css } from 'styled-components';
 
 const FONTS_MAIN = css`
-  font-family: Optima, Verdana, Arial, Times;
+  font-family: Optima, Verdana, Arial, sans-serif;
 `;
 
 const FONTS_SECONDARY = css`
-  font-family: Baskerville, Optima, Arial, Times;
+  font-family: Baskerville, Optima, Arial, sans-serif;
```
Comment (hieronymus-jodi) [inline client/src/constants/Fonts.tsx]: I looked at all of the fonts and I think we should swap out Baskerville for another sans-serif font. Baskerville is the only serif font in these and it sticks out

File: client/src/pages/About.tsx [hunk 4]
```diff
@@ -173,7 +157,7 @@ function About() {
           <Box>
             <H2>Jodi A. Hieronymus</H2>
             <AboutUs>
-              <Img src={jodiPic} alt="Photo of Jodi" />
+              <Img src={jodiPic} alt="Photo of Jodi and Oster the bird" />
```
Comment (hieronymus-jodi) [inline client/src/pages/About.tsx:160]: LOL thank u <3

File: client/package.json [hunk 5]
```diff
@@ -4,6 +4,7 @@
   "proxy": "http://localhost:5000",
   "private": true,
   "dependencies": {
+    "@react-navigation/native-stack": "^6.9.9",
```
Comment (hieronymus-jodi) [inline client/package.json]: Did you use this? Looking through your code looks like you used `useNavigate` instead. If you didn't use this dependency, please remove it

File: client/src/components/simple-components/ButtonsLinks.tsx (hunk 2 above)
Comment (marden-erin) [reply client/src/components/simple-components/ButtonsLinks.tsx]: lol i have no idea why it looks like that it didn't change after i ran prettier, but i fixed it

File: client/src/constants/Fonts.tsx (hunk 3 above)
Comment (marden-erin) [reply client/src/constants/Fonts.tsx]: i changed the font to century gothic which is a sans-serif and i think im in love

**Comment by hieronymus-jodi** [comment]
beautiful ily!!

**Review by hieronymus-jodi — APPROVED** [review]

**Pull request merged**